thoth/graph_metrics_exporter/common.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Comment:
`configuration.py` and `common.py` files can be merged into a single `app.py`. I don't see any reason of having them - they just make the whole application complex.

Reply:
same as above, there will be other two files for `graph_table_bloat_data.py` and `graph_index_bloat_data.py`, each corresponding to other tasks in the cronworkflow